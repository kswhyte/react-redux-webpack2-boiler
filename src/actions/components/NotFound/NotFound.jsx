import React from 'react';

const NotFound = () => {
  return (
    <div className="not-found-wrapper">
      <h1>Not Found!!!</h1>
    </div>
  );
};

NotFound.propTypes = {};

export default NotFound;
