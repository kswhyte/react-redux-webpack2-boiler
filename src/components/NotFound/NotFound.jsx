import React from 'react';

const NotFound = () => {
    return (
        <div className="not-found-block">
            <h1>Not Found!!!</h1>
        </div>
    )
}

NotFound.propTypes = {};

export default NotFound;