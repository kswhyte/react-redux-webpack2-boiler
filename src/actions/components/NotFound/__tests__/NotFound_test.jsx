import React from 'react';
// import { storiesOf, action, linkTo } from '@kadira/storybook';
import NotFound from '../NotFound';

// import { specs, describe, it } from 'storybook-addon-specifications';

import { storiesOf, describe, it, specs } from '../../../../.storybook/facade';

import { StaticRouter } from 'react-router';

import { mount } from 'enzyme';
import expect from 'expect';

const context = {};

const stories = storiesOf('NotFound', module);

stories.add('Story', () => {
  const notFoundStory = (
    <StaticRouter context={context}>
      <NotFound />
    </StaticRouter>
  );

  specs(() =>
    describe('Show NotFound', () => {
      it('Should render the NotFound component without crashing', () => {
        let output = mount(notFoundStory);
        expect(output.find('.not-found-wrapper').length).toEqual(1);
      });
    })
  );
  return notFoundStory;
});
