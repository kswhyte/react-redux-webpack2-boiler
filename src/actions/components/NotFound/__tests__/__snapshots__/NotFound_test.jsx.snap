// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Story 1`] = `
<div
  className="not-found-wrapper"
>
  <h1>
    Not Found!!!
  </h1>
</div>
`;
